scripts/inventory.gd

```diff
@@ -2,23 +2,24 @@ extends Control
 onready var itemlist:ItemList = $CanvasLayer/ItemList
 var items:Dictionary = {
 	"Knife": {
-	'picture': 'res://assets/items/knife.png',
-	"description": 'A standard knife',
+		'picture': 'res://assets/items/knife.png',
+		"description": 'A standard knife',
 	},
 	"Hell Knife": {
-	'picture': 'res://assets/items/hell_knife.png',
-	"description": 'The most powerful knife',
+		'picture': 'res://assets/items/hell_knife.png',
+		"description": 'The most powerful knife',
 	},
 	"Wooden Knife": {
-	"description": 'A knife made out of wood, not very effective...',
-	'picture': 'res://assets/items/wooden_knife.png',
+		"description": 'A knife made out of wood, not very effective...',
+		'picture': 'res://assets/items/wooden_knife.png',
 	}
 }
 func _ready():
 	$CanvasLayer.visible = false #<---- I can't do this in editor, so I have to do it via code.
-	additem('Knife')
-	additem('Wooden Knife')
-	additem('Hell Knife')
+	var count = 0
+	while not count == 3:
```

Comment:
Are we only going to have 3 items or were we planning to have more? What if we wanted to add more? And if we did add more we would have to keep coming here to modify this code right? Is there a better way of doing this?

Reply:
There are gonna only be 3 item slots, and this is just placeholder code, so..